sdk/eventhubs/azure-messaging-eventhubs/migration-guide.md

```diff
@@ -405,27 +296,101 @@ private static void onEvent(EventContext eventContext) {
     System.out.println("Contents: " + new String(event.getBody(), StandardCharsets.UTF_8));
 }
 ```
-#### V3 Checkpoints
-In order to align with the goal of supporting cross-language checkpoints and a more efficient means of tracking 
-partition ownership, V5 Event Processor Client does not consider or apply checkpoints created with the legacy Event
-Processor Host family of types. To migrate the checkpoints created by the V3 Event Processor Host, the new Event 
-Processor Client provides an option to do a one-time initialization of checkpoints as shown in the sample below.
+
+### Receive events from a single partition
+
+In v3, events were received by creating a `PartitionReceiver` and invoking `receive(int)` multiple times to receive
+events up to a certain number.
+
+In v5, [project Reactor][project-reactor] is used, so events can be streamed as they come in without having to use a
```

Comment:
We cannot receive events from single partition in EventHubConsumerClient?

Reply:
We can and since we are examining production scenario migrations, I opted to use something that was the same, which we would expect in production.